packages/secondary-nav/src/SecondaryNav.stories.tsx

```diff
@@ -44,22 +44,3 @@ DarkAltVariant.args = {
 	activePath: '#code',
 	links: exampleLinks,
 };
-
-export const Overflow = Template.bind({});
```

Comment:
We don't want to encourage this many items in a secondary navigation component so this story has been removed